skyportal/facility_apis/sedm.py

```diff
@@ -392,10 +430,8 @@ def prepare_payload(payload, existing_payload=None):
                             "advanced": {"enum": [True]},
                             "exposure_time": {
                                 "title": "Exposure Time (Photometry) [s]",
-                                "type": "number",
-                                "default": -1,
-                                "minimum": -1,
-                                "maximum": 3600,
```

Comment:
I believe Don had suggested this maximum before. We can remove it here, but if @scizen9 wants to keep that limit (or in general if we want to avoid number lower than -1), we should add some verification somewhere right after the one you added line 184.

Reply:
Added!

Reply:
Currently, the sedm watcher code just checks if exptime < 0, so any negative number will work.